src/openxr/OpenXRConfig.cpp

```diff
@@ -45,6 +51,44 @@ bool OpenXRConfig::keep_3d_linear() const {
 	}
 }
 
+int OpenXRConfig::get_view_config_type() const {
+	if (openxr_api == NULL) {
+		return 1;
+	} else {
+		switch (openxr_api->get_view_configuration_type()) {
+			case XR_VIEW_CONFIGURATION_TYPE_PRIMARY_MONO: {
+				return 0;
+			}; break;
+			case XR_VIEW_CONFIGURATION_TYPE_PRIMARY_STEREO: {
+				return 1;
+			}; break;
+			default: {
+				// we don't support the others (yet)
+				return 1;
+			}; break;
+		}
+	}
+}
+
+void OpenXRConfig::set_view_config_type(const int p_config_type) {
+	// TODO may need to add something here that this is read only after initialisation
+	if (openxr_api == NULL) {
+		Godot::print("OpenXR object wasn't constructed.");
+	} else {
+		switch (p_config_type) {
+			case 0: {
+				openxr_api->set_view_configuration_type(XR_VIEW_CONFIGURATION_TYPE_PRIMARY_MONO);
+			}; break;
+			case 1: {
+				openxr_api->set_view_configuration_type(XR_VIEW_CONFIGURATION_TYPE_PRIMARY_STEREO);
+			}; break;
+			default: {
```

Comment:
Similar comment here about merging `default` with `case 1`.

Reply:
As said, I like the explicitness here. Especially once we get more options it becomes clear what behavior is designed, but it's really nothing more then personal preference :)